lib/nexmo_developer/app/views/layouts/partials/_header.html.erb

```diff
@@ -3,6 +3,7 @@
 
 <div class="Adp-header">
   <header id="header" class="Adp-header__main Vlt-M-plus">
+  <input type="hidden" id="adobe_launch_url" value="<%=ENV['ADOBE_LAUNCH_URL']%>">
```

Comment:
I don't believe this is required.

Reply:
This is corrected, we are reading ADOBE_LAUNCH_PRIMARY_CATEGORY.